include/aspect/boundary_temperature/dynamic_core.h

```diff
@@ -377,15 +377,15 @@ namespace aspect
          *    So that         Q+Qs*dT/dt+Qr+Qg*dR/dt*Ql*dR/dt=0
          * 3. The light component composition X depends on inner core radius (See function get_X() ),
          *    and core solidus may dependent on X as well.
-         *    This becomes a small nonliner problem. Directly iterate through the above three equations doesn't
+         *    This becomes a small nonlinear problem. Directly iterate through the above three equations doesn't
          *    converge well. Alternatively we solve the inner core radius by bisection method.
-         *    A sigle solution between fully liquid and fully solid core is expacted. Otherwise this function will throw exception and terminate.
+         *    A single solution between fully liquid and fully solid core is expected. Otherwise this function will throw exception and terminate.
          *
-         *    At Earth core condition, a inner core is forming at the center of the Earth and surrended by a liquid outter core.
-         *    However, the core solidus is influnced by light components (e.g. S) and its slope is very closed to core adiabatic. So there is an alternative
-         *    scenario that the crystialization happens first at the core mantle boundary instead of at the center, which is called a 'snowing core'
+         *    At Earth core condition, a inner core is forming at the center of the Earth and surrounded by a liquid outer core.
+         *    However, the core solidus is influenced by light components (e.g. S) and its slope is very closed to core adiabatic. So there is an alternative
+         *    scenario that the crystallization happens first at the core mantle boundary instead of at the center, which is called a 'snowing core'
          *    (Stewart, A. J., et al. (2007). "Mars: a new core-crystallization regime." Science 316(5829): 1323-1325.). This also
-         *    provides a valid solution for the slover. So the returning bool is set to true for normal core, and false for 'snowing core'.
+         *    provides a valid solution for the solver. So the returning bool is set to true for normal core, and false for 'snowing core'.
```

Comment:
Oof, that's a lot of bad typos in such a short piece of text. (One wonders who reviewed that, but nothing good usually comes out of asking that question.)